### DISCUSSION TITLE
请求上架android tv 商店

### DISCUSSION THREAD

**Discussion by dwbii** [2dust/v2rayNG#3150] [General]
在安卓TV商店搜不到此应用，但手动安装后能正常运行并代理

希望能在安卓TV商店直接安装
![屏幕截图 2024-05-23 083921](https://github.com/2dust/v2rayNG/assets/18597768/55dda8e8-93aa-4a11-b950-3e163643f63e)

**Comment by 2dust** [comment]
因为上架的一些限制，暂时无法处理；请先从github下载
